release/__init__.py

```diff
@@ -398,26 +399,27 @@ def make_channel_artifacts(metadata):
 
         # Add templates for the default variant.
         # Use keyword args to make not matching ordering a loud error around changes.
-        for built_resource in module.do_create(
-                tag=metadata['tag'],
-                build_name=metadata['build_name'],
-                reproducible_artifact_path=metadata['reproducible_artifact_path'],
-                commit=metadata['commit'],
-                variant_arguments=variant_arguments,
-                all_bootstraps=metadata["all_bootstraps"]):
-
-            assert isinstance(built_resource, dict), built_resource
-
-            # Type switch
-            if 'packages' in built_resource:
-                for package in built_resource['packages']:
-                    artifacts.append(get_gen_package_artifact(package))
-            else:
-                assert 'packages' not in built_resource
-                artifacts.append(built_resource)
-
-        # TODO(cmaloney): Check the provider artifacts adhere to the artifact template.
-        artifacts += provider_data.get('artifacts', list())
+        with logger.scope("Creating {} installer".format(module.__name__)):
```

Comment:
Can we change this to "Creating {} deploy tools" (see: https://github.com/dcos/dcos/pull/978)